pghoard/webserver.py

```diff
@@ -427,16 +427,25 @@ def _try_save_and_verify_restored_file(self, filetype, filename, prefetch_target
                     os.unlink(prefetch_target_path)
             return e
 
+    @staticmethod
+    def _validate_target_path(config, filename):
+        xlog_file = Path(os.path.relpath(filename))
+        data_dir = Path(os.path.relpath(config["pg_data_directory"]))
+        if data_dir not in xlog_file.parents:
```

Comment:
I don't really understand why you use relpath for the filename, especially according to the current directory ?

Reply:
Added comments in the code on why we expect the `target_path` to be an absolute path